[2024-03-25T03:34:54.810+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Book-Recommendation-DAG.install_pip_packages_task manual__2024-03-25T03:34:53.060749+00:00 [queued]>
[2024-03-25T03:34:54.817+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Book-Recommendation-DAG.install_pip_packages_task manual__2024-03-25T03:34:53.060749+00:00 [queued]>
[2024-03-25T03:34:54.817+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 3
[2024-03-25T03:34:54.826+0000] {taskinstance.py:2217} INFO - Executing <Task(BashOperator): install_pip_packages_task> on 2024-03-25 03:34:53.060749+00:00
[2024-03-25T03:34:54.831+0000] {standard_task_runner.py:60} INFO - Started process 3806 to run task
[2024-03-25T03:34:54.834+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Book-Recommendation-DAG', 'install_pip_packages_task', 'manual__2024-03-25T03:34:53.060749+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/book-recommendation-dag.py', '--cfg-path', '/tmp/tmpxqrzoctu']
[2024-03-25T03:34:54.837+0000] {standard_task_runner.py:88} INFO - Job 61: Subtask install_pip_packages_task
[2024-03-25T03:34:54.879+0000] {task_command.py:423} INFO - Running <TaskInstance: Book-Recommendation-DAG.install_pip_packages_task manual__2024-03-25T03:34:53.060749+00:00 [running]> on host 9654d883592e
[2024-03-25T03:34:54.956+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='iamraphson' AIRFLOW_CTX_DAG_ID='Book-Recommendation-DAG' AIRFLOW_CTX_TASK_ID='install_pip_packages_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-25T03:34:53.060749+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-25T03:34:53.060749+00:00'
[2024-03-25T03:34:54.958+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-25T03:34:54.959+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'pip install --user kaggle']
[2024-03-25T03:34:54.972+0000] {subprocess.py:86} INFO - Output:
[2024-03-25T03:34:55.523+0000] {subprocess.py:93} INFO - Collecting kaggle
[2024-03-25T03:34:55.524+0000] {subprocess.py:93} INFO -   Using cached kaggle-1.6.6-py3-none-any.whl
[2024-03-25T03:34:55.531+0000] {subprocess.py:93} INFO - Requirement already satisfied: six>=1.10 in /home/***/.local/lib/python3.8/site-packages (from kaggle) (1.16.0)
[2024-03-25T03:34:55.532+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi in /home/***/.local/lib/python3.8/site-packages (from kaggle) (2024.2.2)
[2024-03-25T03:34:55.532+0000] {subprocess.py:93} INFO - Requirement already satisfied: python-dateutil in /home/***/.local/lib/python3.8/site-packages (from kaggle) (2.9.0.post0)
[2024-03-25T03:34:55.532+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests in /home/***/.local/lib/python3.8/site-packages (from kaggle) (2.31.0)
[2024-03-25T03:34:55.532+0000] {subprocess.py:93} INFO - Requirement already satisfied: tqdm in /home/***/.local/lib/python3.8/site-packages (from kaggle) (4.66.2)
[2024-03-25T03:34:55.533+0000] {subprocess.py:93} INFO - Requirement already satisfied: python-slugify in /home/***/.local/lib/python3.8/site-packages (from kaggle) (8.0.4)
[2024-03-25T03:34:55.533+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3 in /home/***/.local/lib/python3.8/site-packages (from kaggle) (1.26.18)
[2024-03-25T03:34:55.533+0000] {subprocess.py:93} INFO - Requirement already satisfied: bleach in /home/***/.local/lib/python3.8/site-packages (from kaggle) (6.1.0)
[2024-03-25T03:34:55.536+0000] {subprocess.py:93} INFO - Requirement already satisfied: webencodings in /home/***/.local/lib/python3.8/site-packages (from bleach->kaggle) (0.5.1)
[2024-03-25T03:34:55.539+0000] {subprocess.py:93} INFO - Requirement already satisfied: text-unidecode>=1.3 in /home/***/.local/lib/python3.8/site-packages (from python-slugify->kaggle) (1.3)
[2024-03-25T03:34:55.543+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests->kaggle) (3.3.2)
[2024-03-25T03:34:55.543+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests->kaggle) (3.6)
[2024-03-25T03:34:57.174+0000] {subprocess.py:93} INFO - Installing collected packages: kaggle
[2024-03-25T03:34:57.256+0000] {subprocess.py:93} INFO - Successfully installed kaggle-1.6.6
[2024-03-25T03:34:57.334+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-25T03:34:57.352+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=Book-Recommendation-DAG, task_id=install_pip_packages_task, execution_date=20240325T033453, start_date=20240325T033454, end_date=20240325T033457
[2024-03-25T03:34:57.396+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-25T03:34:57.413+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
